build-version.sh

```diff
@@ -30,6 +30,20 @@ if [ "$TARGET_ARCH" == "x86_64-unknown-linux-musl" ]; then
 
     CARGO_BIN_DIR="${TEMPDIR}/cargo/bin"
     CRATES2_JSON_PATH="${TEMPDIR}/cargo/.crates2.json"
+elif [ "$TARGET_ARCH" == "aarch64-unknown-linux-gnu" ]; then
+    wget $(curl https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//') -O zig
+    mkdir zigfolder
+    tar -xf ./zig -C zigfolder --strip-components 1
+    rm zig
+    sudo mv zigfolder/zig /usr/local/bin
+    rustup target add "$TARGET_ARCH"
+    echo "[target.aarch64-unknown-linux-gnu]" >> ~/.cargo/config
+    echo '["linker = "./zig-aarch64.sh"]' >> ~/.cargo.config
+
+    CARGO_PROFILE_RELEASE_CODEGEN_UNITS="1" CARGO_PROFILE_RELEASE_LTO="fat" OPENSSL_STATIC=1 cargo install "$CRATE" CC=./zig-aarch64.sh --version "$VERSION" --target "$TARGET_ARCH"
```

Comment:
According to the [CI](https://github.com/cargo-bins/cargo-quickinstall/actions/runs/3915594303/jobs/6693920659#step:3:960), cargo still uses `ld`.

Reply:
weird...